addons/mail/static/src/new/composer/composer_suggested_recipient_list.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.composer_suggested_recipients_list" owl="1">
+        <t t-if="props.hasFollowers and props.thread.composer.type !== 'note'">
```

Comment:
`props.hasFollowers` is only used to guard showing of suggested recipients list.
It feels more appropriate to not show this component instead of passing props and in the end not show the content